app/src/containers/map/Share.jsx

```diff
@@ -1,7 +1,11 @@
 import { connect } from 'react-redux';
```

Comment:
I thought that containers started with a lowercase letter, but maybe it's just from legacy code? Can we decide on this?